src/librustdoc/html/format.rs

```diff
@@ -1327,15 +1327,15 @@ impl clean::GenericArg {
     }
 }
 
-crate fn display_fn(f: impl FnOnce(&mut fmt::Formatter<'_>) -> fmt::Result) -> impl fmt::Display {
+crate fn display_fn(f: impl Fn(&mut fmt::Formatter<'_>) -> fmt::Result) -> impl fmt::Display {
     WithFormatter(Cell::new(Some(f)))
 }
 
-struct WithFormatter<F>(Cell<Option<F>>);
+struct WithFormatter<F: Fn(&mut fmt::Formatter<'_>) -> fmt::Result>(Cell<Option<F>>);
```

Comment:
This change is not necessary, but it seemed like a good idea.

Reply:
I think you're just duplicating a trait bound, so if it has to be changed in the future, it'll have to be changed twice. I personally think this is over-specification, but no big deal here.

Reply:
I agree, I don't think this change is necessary.